apps/vaults/contexts/useSolver.tsx

```diff
@@ -0,0 +1,131 @@
+import React, {createContext, useCallback, useContext, useEffect, useMemo, useState} from 'react';
+import {useActionFlow} from '@vaults/contexts/useActionFlow';
+import {useSolverChainCoin} from '@vaults/hooks/useSolverChainCoin';
+import {useSolverCowswap} from '@vaults/hooks/useSolverCowswap';
+import {useSolverPartnerContract} from '@vaults/hooks/useSolverPartnerContract';
+import {useSolverVanilla} from '@vaults/hooks/useSolverVanilla';
+import {useWeb3} from '@yearn-finance/web-lib/contexts/useWeb3';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+import performBatchedUpdates from '@yearn-finance/web-lib/utils/performBatchedUpdates';
+import {DefaultTNormalizedBN} from '@common/utils';
+
+import type {TNormalizedBN} from '@common/types/types';
+import type {TWithSolver} from '@vaults/types/solvers';
+
+export enum	Solvers {
```

Comment:
```suggestion
export enum	Solver {
```
nit: Common pattern is to have enums singular, refer [here](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)